src/Microsoft.ML.Data/Commands/SaveDataCommand.cs

```diff
@@ -129,11 +130,10 @@ private void RunCore(IChannel ch)
 
             if (!string.IsNullOrWhiteSpace(Args.Columns))
             {
-                var args = new ChooseColumnsTransform.Arguments();
-                args.Column = Args.Columns
-                    .Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(s => new ChooseColumnsTransform.Column() { Name = s }).ToArray();
-                if (Utils.Size(args.Column) > 0)
-                    data = new ChooseColumnsTransform(Host, args, data);
+                var keepColumns = Args.Columns
+                    .Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).ToArray();
+                if (keepColumns.Length > 0)
```

Comment:
>Length [](start = 32, length = 6)

Does Split always return non-null? Otherwise we should use Utils.Size(keepColumns) instead of keepColumns.Length.